documentation/docs/30-advanced/25-errors.md

```diff
@@ -95,14 +95,21 @@ import * as Sentry from '@sentry/sveltekit';
 Sentry.init({/*...*/})
 
 /** @type {import('@sveltejs/kit').HandleServerError} */
-export function handleError({ error, event }) {
-	// example integration with https://sentry.io/
-	Sentry.captureException(error, { extra: { event } });
-
-	return {
-		message: 'Whoops!',
-		code: error?.code ?? 'UNKNOWN'
-	};
+export function handleError({ error, event, status, message }) {
+	if (status !== 500) {
+		return {
+			message, // safe to forward
+			code: 'UNKNOWN'
+		};
+	} else {
+		// example integration with https://sentry.io/
+		Sentry.captureException(error, { extra: { event } });
+
+		return {
+			message: 'Whoops!',
+			code: error?.code ?? 'UNKNOWN'
+		};
+	}
```

Comment:
I'd probably write this as:
```suggestion
	if (status >= 500) {
		// example integration with https://sentry.io/
		Sentry.captureException(error, { extra: { event } });

		return {
			message: 'Whoops!',
			code: error?.code ?? 'UNKNOWN'
		};
	}
	return {
		message, // safe to forward
		code: 'UNKNOWN'
	};
```

Reply:
the `>=` is unnecessary — SvelteKit will never throw a 5xx error itself, so the only way you could get a 5xx error other than 500 is by using `error(...)`, in which case `handleError` wouldn't be involved